elements/phaseindicator.lua

```diff
@@ -7,16 +7,27 @@ Toggles the visibility of an indicator based on the unit's phasing relative to t
 
 PhaseIndicator - Any UI widget.
 
+## Sub-Widgets
+
+Icon - A `Texture` to represent the phased status.
+
 ## Notes
 
 A default texture will be applied if the widget is a Texture and doesn't have a texture or a color set.
+OnEnter and OnLeave script handlers will be set to display a Tooltip if the widget is mouse enabled and does not have
+OnEnter and/or OnLeave handlers.
```

Comment:
I'd leave a note about the optional frame vs texture, specifically as the first note here.

Reply:
I would leave it like it is so we don't have to support it later. The default case is Blizzard like or better - so tooltip it is. The backwards compat is more of a side effect than anything else.